src/code/style.css

```diff
@@ -442,4 +442,35 @@ table td {
 #save-btn:active {
     transform: scale(0.9);
     transition-duration: 100ms;
-}
\ No newline at end of file
+}
+
+/* ==========================Link Validation====================== */
+
+#linkInputContainer {
+    position: relative;
+    margin-bottom: 20px;
+}
+
+#linkInput {
+    width: 300px;
+    padding: 5px;
+    border-bottom: 2px solid #7029ff;
+}
+
+#errorText {
+    position: absolute;
+    bottom: -20px;
+    left: 0;
+    color: red;
+    display: none;
+}
+
+#previewButton {
+    cursor: not-allowed;
+    opacity: 0.5;
+}
+
+#previewButton.enabled {
+    cursor: pointer;
+    opacity: 1;
+}
```

Comment:
The ids for which you've designed the css, these IDs doesn't exist in `index.html`, kindly, read the `index.html` file and do changes in CSS appropriately.